abci/README.md

```diff
@@ -19,8 +18,8 @@ To get up and running quickly, see the [getting started guide](../docs/app-dev/g
 
 A detailed description of the ABCI methods and message types is contained in:
 
-- [The main spec](https://github.com/tendermint/tendermint/blob/v0.37.x/spec/abci/abci.md)
-- [A protobuf file](./types/types.proto)
+- [The v0.37.x spec](https://github.com/cometbft/cometbft/blob/main/spec/abci/README.md)
```

Comment:
```suggestion
- [The v0.37.x spec](https://github.com/cometbft/cometbft/blob/v0.37.x/spec/abci/README.md)
```
Or maybe we keep it pointing to main until 0.38 is released? Although for a release it should point to 0.37?

Reply:
The title mentions 0.37.x, my fault.